logger_core/tests/test_logger.rs

```diff
@@ -9,7 +9,10 @@ use test_env_helpers::*;
 mod tests {
     use logger_core::{init, log_debug, log_trace};
     use rand::{distributions::Alphanumeric, Rng};
-    use std::fs::{read_dir, read_to_string, remove_dir_all};
+    use std::{
+        fs::{read_dir, read_to_string, remove_dir_all},
+        path::Path,
+    };
     const FILE_DIRECTORY: &str = "glide-logs";
```

Comment:
You have it here :) use it

Reply:
I created this constant in the lib.rs file since I don't think I should import it from the test file